src/encoder.rs

```diff
@@ -91,34 +91,34 @@ fn encode_bundle(bundle: &OscBundle) -> Result<Vec<u8>> {
     Ok(bundle_bytes)
 }
 
-fn encode_arg(arg: &OscType) -> Result<(Option<Vec<u8>>, char)> {
+fn encode_arg(arg: &OscType) -> Result<(Option<Vec<u8>>, String)> {
     match *arg {
         OscType::Int(ref x) => {
             let mut bytes = vec![0u8; 4];
             BigEndian::write_i32(&mut bytes, *x);
-            Ok((Some(bytes), 'i'))
+            Ok((Some(bytes), 'i'.to_string()))
```

Comment:
You can replace the `.to_string()` calls by wrapping the chars in double quotes.

Reply:
Ok, it's not that easy but `"x".into()` works.